Zend/zend_portability.h

```diff
@@ -442,6 +442,20 @@ char *alloca();
 
 #define ZEND_NAN NAN
 
+#if defined(__cplusplus) && __cplusplus >= 201103L
+	extern "C++" {
+# include <cmath>
+# define zend_isnan std::isnan
+# define zend_isinf std::isinf
+# define zend_finite std::isfinite
+	}
```

Comment:
For clarity, it's better to put the `}` right after the include, not the defines.